trashtalk/seeclickfix.py

```diff
@@ -8,6 +9,9 @@
 # Switching to live page requires removing 'test' from url and creating account for live page
 HEADER = {"Content-type": "application/json"}
 BASE_CALL = "https://test.seeclickfix.com/api/v2/issues"
+ADMIN_USERNAME = ""
```

Comment:
Lets move these to `constants.py` and rename explicitly:
```
SCF_HEADER
SCF_BASE_URL
SCF_USERNAME
SCF_PASSWORD
SCF_CLEANUP_BASE_URL
```

Reply:
**Username** and **Password** must be protected. Also, **Cleanup_Base_URL** will change when we move from *local* to *web* serving. Consequently, I already placed these three variables in **settings.py/dev.cfg**. To keep all the variables together, I'll move **Header** and **Base_Url** there as well.